app/lib/tariff_synchronizer/taric_update_downloader.rb

```diff
@@ -62,5 +60,17 @@ def update_or_create(state, file_name)
     def log_request_to_taric_update
       instrument('get_taric_update_name.tariff_synchronizer', date: date, url: url)
     end
+
+    def date_api_url
+      sprintf(taric_query_url_template, host: host, date: date.strftime('%Y%m%d'))
+    end
+
+    def file_api_urls
```

Comment:
I cannot find a test for this, I think we should write one.

Reply:
Multiple times, actually